ROADMAP.md

```diff
@@ -0,0 +1,58 @@
+# go-libp2p roadmap Q4’22/Q1’23
```

Comment:
Great write up! 
A general thought is that as libp2p is getting more widely deployed maybe we also need to think about beef up more security features.  May or may not fit in to the next 6 months time frame.
For example behavior based security. The peer is usually not backed by a root of trust, so the traditional identity and chain of trust based approach does not work in this scenario. We may need to consider new approaches to enforce security.

Reply:
@julian88110 Do you have anything specific in mind?
Would that belong into the go-libp2p roadmap, or would that first and foremost be an item for the general libp2p roadmap (https://github.com/libp2p/specs/blob/master/ROADMAP.md)?

Reply:
Yes this should belong to the general roadmap for libp2p. I have some thoughts, may take some time to write it up after the muxer thing is done.